x-pack/plugins/security_solution/public/resolver/test_utilities/extend_jest.ts

```diff
@@ -17,6 +17,7 @@ declare global {
   namespace jest {
     interface Matchers<R, T> {
       toYieldEqualTo(expectedYield: T extends AsyncIterable<infer E> ? E : never): Promise<R>;
+      toYieldObjectEqualTo(expectedYield: unknown): Promise<R>;
```

Comment:
:grey_question: Not sure about making this even more complicated. What does it do that toYieldEqualTo doesn't? Would be helpful to read in a comment here.

Reply:
yes